src/api/bodhi_utils.js

```diff
@@ -0,0 +1,13 @@
+const { getInstance } = require('../qclient');
+
+const BodhiUtils = {
```

Comment:
rename to `Util`. this is supposed to correlate to `Util` section in qtum client help. I guess `QtumUtils` makes more sense since we already have a `utils.js`.